### PR TITLE
Add URL to failed builds consolidate failure messages in one place

### PR DESCRIPTION
**Pull request by edpaget**
Use a post failure step instead of notifying in a try catch. Include a link to the Blue Ocean URL.

### CODE REVIEW

**Comment by edpaget** [comment]
This at least includes a link to the failed build on Jenkins so we can see what happened. Still working on seeing if its possible to grab test failures.

**Comment by edpaget** [comment]
Was able to add junit reporting of the unit tests, so can now do some fancier reports around failed tests in the future.

**Review by wyattwalter — APPROVED** [review]

File: Jenkinsfile [hunk 1]
```diff
@@ -42,12 +42,13 @@ def buildDetails = { vars ->
   """.stripIndent()
 }
 
-def notify = { message, color='good' ->
+def notify = { ->
   if (env.BRANCH_NAME == devBranch ||
       env.BRANCH_NAME == stagingBranch ||
       env.BRANCH_NAME == prodBranch) {
+    message = "vets-website ${env.BRANCH_NAME} branch CI failed. |${env.RUN_DISPLAY_URL}".stripMargin()
```
Comment (wyattwalter) [inline Jenkinsfile]: neato, did not know about RUN_DISPLAY_URL

Comment (edpaget) [reply Jenkinsfile]: Yeah it links the Blue Ocean UI which I think is nicer for the vets-website build. Turns out the Jenkins docs around it are still not great.

**Pull request merged**